addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php

```diff
@@ -286,7 +285,8 @@ public function index($category = null, $city = null)
             $allCats = true;
         }
 
-        $cFArray = array();
+        $cFArray = $checkboxes = $topfields = $selectDropdown = $selectRange = $selectImage = $ranges = $radio = array();
```

Comment:
Any reason ?

Reply:
otherwise if custom fields is not install gives error because variable not defined